src/averageAges.js

```diff
@@ -37,7 +43,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(({ sex, name }) => sex === 'f'
+  && (withChildren
+    ? people.some(person => person.mother === name)
+    : true
+  ));
```

Comment:
please check the indents
```suggestion
  const women = people.filter(({ sex, name }) => sex === 'f'
     && (withChildren
       ? people.some(person => person.mother === name)
       : true
    ));
```